src/api/app/assets/javascripts/webui2/staging_workflow.js

```diff
@@ -7,3 +7,21 @@ function setSpinnersForDeletion() { // jshint ignore:line
     $(this).find('.delete-spinner').addClass('d-none');
   });
 }
+
+function autocompleteStagingManagersGroup() { // jshint ignore:line
```

Comment:
Maybe we should disable this cop. We are adding the `ignore:line` everywhere! :weary: (not necessarily in this PR)

Reply:
If we want to disable the cop is something for another PR. Please open one if you want :smile: